src/code/views/app-view.coffee

```diff
@@ -0,0 +1,67 @@
+LinkView    = React.createFactory require './link-view'
+NodeWell    = React.createFactory require './node-well-view'
+NodeEditView= React.createFactory require './node-edit-view'
+LinkEditView= React.createFactory require './link-edit-view'
+StatusMenu  = React.createFactory require './status-menu-view'
```

Comment:
This was new in React 0.12 and is related to them moving to a pure class based system.  

A good explanation is here: https://facebook.github.io/react/blog/2014/10/14/introducing-react-elements.html

In 0.13 passing a function generated by createClass in another components render method works but generates a console warning.  I added the createFactory calls to suppress the warning messages.  The JSX compiler automatically adds the createFactory() wrapper which is why you didn't see the warnings before.